openshift/README.md

```diff
@@ -184,7 +184,7 @@ oc -n k8vopl-<env> rollout latest dc/<buildname>-master
 As of this time, we do not automatically clean up resources generated by a Pull Request once it has been accepted and merged in. This is still a manual process. Our PR deployments are all named in the format "pr-###", where the ### is the number of the specific PR. In order to clear all resources for a specific PR, run the following two commands to delete all relevant resources from the Openshift project (replacing `PRNUMBER` with the appropriate number):
 
 ```sh
-oc delete all -n k8vopl-dev --selector app=getok-pr-<PRNUMBER>
+oc delete all,nsp -n k8vopl-dev --selector app=getok-pr-<PRNUMBER>
```

Comment:
Think you need to add CHES_TOKEN_ENDPOINT etc to the config map instructions.

Reply:
Yes missed these, updated. Thanks